### ISSUE TITLE
Intuitive way to save the survey mid-recording

### ISSUE THREAD

**Issue by spado** [Esri/Survey123Community#304]
This is just a suggestion for future updates.

I've developed surveys for old men in their 50s/60s to use.  They aren't very tech-savvy and I have started to observe how they use Survey123 in the field.  I am impressed with the many ways they were able to make an iPhone shut down and how often they are able to get distracted mid-survey.   I've encouraged them to save the survey regularly so they don't lose much data.  However, the only way to save is by closing off the survey and then re-opening.  

It would be awesome if there was some sort of quick save function in future versions.

**Comment by Marie-Lou** [comment]
I agree - a quick save function, or even an auto save will be awesome.  

My users don't necessarily get distracted, but they have to move about on site, and do more things than just completing a survey in one go - auto save will make their life (and mine....) much easier.

**Comment by RooPitt** [comment]
+1
I ran a pilot yesterday with mostly tech-savvy 20-somethings and had the same problem...I don't know how they managed it but around 5 out of 30 complained that they'd lost data after accidentally shutting down the app... This would be a great enhancement

**Comment by travisbutcher** [comment]
👍 
We have had reports of surveys dropping if something happens during collection.  Some forms may take a few hours of capturing data (observations over time) before they are ready to be saved or submitted.  Having Survey123 autosave during this process would make these data safer over these long transactions.

**Comment by survey123** [comment]
Flagging as planned. We are researching how to do this. Thanks all for your feedback.
